four-principles.md

```diff
@@ -0,0 +1,10 @@
+Operate First regroups a few concepts, relying on standard definitions of Open, Open Source, and Open Work:
+
+OPERATE:
+We build software optimized to run as a service and allow others to run it as a service easily, including documenting how we operate the cloud environment itself.
+
+OPEN: We develop open source practices, tooling, and documentation that allows us to build and manage our software, incorporate lessons we’ve learned from operations back into code, automate wherever possible, and contribute to open source communities.
```

Comment:
Suggestion offered and accepted from @chirino that this be changes so it's clear we mean everyone, everywhere:
```suggestion
OPEN: We develop open source practices, tooling, and documentation that allows anyone to build and manage our software, incorporate lessons we’ve learned from operations back into code, automate wherever possible, and contribute to open source communities.
```